.github/workflows/linux.yml

```
@@ -13,7 +13,7 @@ concurrency:
 
 jobs:
   linux:
-    name: ${{ matrix.distro }}
+    name: ${{ matrix.distro }}${{ matrix.platform != 'linux/amd64' && format(' ({0})', matrix.platform) || '' }}
```

Redundant space before `({0...)`.

---

```suggestion
    name: ${{ matrix.distro }}${{ matrix.platform != 'linux/amd64' && format('({0})', matrix.platform) || '' }}
```

This space? It's been added intentionally.

---

Ah! I didn't see the final github job names!